app/src/matchmaking/match/match.cpp

```diff
@@ -52,7 +52,9 @@ void Match::addMoveData(const Player& player, int64_t measured_time_ms, bool leg
         ss << (move_data.score >= 0 ? '+' : '-');
         ss << std::fixed << std::setprecision(2) << (float(std::abs(move_data.score)) / 100);
     } else if (score_type == engine::ScoreType::MATE) {
-        ss << (move_data.score > 0 ? "+M" : "-M") << std::to_string(std::abs(move_data.score));
+        uint64_t plies = move_data.score > 0 ? std::abs(move_data.score) * 2 - 1
```

Comment:
Abs not needed here I think. Similarly below.

Reply:
Right, its always positive, fixed 👍